app/models/street.js

```diff
@@ -26,14 +26,14 @@ streetSchema.add({
 })
 
 streetSchema.pre('save', function (next) {
-  var now = new Date()
+  const now = new Date()
   this.updated_at = now
   this.created_at = this.created_at || now
   next()
 })
 
 streetSchema.methods.asJson = function (cb) {
-  var json = {
+  let json = {
```

Comment:
It doesn't look like `json` is ever reassigned.

Objects whose properties are assigned or reassigned values later can still be defined as `const`, because the variable's reference to the object itself never changes. An object defined as a `const` variable does _not_ mean the object is immutable.